cmp/grid/Grid.scss

```diff
@@ -209,28 +209,28 @@
   //------------------------
   // Tree grid styling modes
   //------------------------
-  &.xh-grid--tree-style-highlights {
-    div[class*=" ag-row-group"],
-    div[class^="ag-row-group"] {
-      &:not(.ag-row-selected),
-      &.ag-row.ag-row-odd:not(.ag-full-width-row):not(.ag-row-selected) {
-        background-color: var(--xh-grid-tree-group-bg);
+  @for $i from 0 to 20 {
+    &.xh-grid--tree-style-highlights-level-#{$i} {
+      .ag-row:not(.ag-row-level-#{$i}) {
+        &:not(.ag-row-selected),
+        &.ag-row.ag-row-odd:not(.ag-full-width-row):not(.ag-row-selected) {
+          background-color: var(--xh-grid-tree-group-bg);
+        }
       }
-    }
 
-    .xh-ag-grid--show-hover {
-      div[class*=" ag-row-group"],
-      div[class^="ag-row-group"] {
-        &.ag-row.ag-row-hover:not(.ag-full-width-row):not(.ag-row-selected) {
-          background-color: var(--xh-grid-bg-hover);
+      .xh-ag-grid--show-hover {
+        div[class*=" ag-row-group"],
+        div[class^="ag-row-group"] {
+          &.ag-row.ag-row-hover:not(.ag-full-width-row):not(.ag-row-selected) {
+            background-color: var(--xh-grid-bg-hover);
+          }
         }
       }
     }
   }
 
   &.xh-grid--tree-style-borders {
-    .ag-row-level-0[class*=" ag-row-group"],
-    .ag-row-level-0[class^="ag-row-group"], {
+    .ag-row-level-0 {
```

Comment:
Wondering if there was some other reason why we needed those extra selectors here - I think there must have been... Would want to check if any other rows we're not accounting for could start matching (e.g. summary rows?)

Reply:
Just tested and summary rows are not taking on any new styles.